src/statusnotifieritem/statusnotifieritem.cpp

```diff
@@ -62,17 +68,17 @@ StatusNotifierItem::StatusNotifierItem(QString id, QObject *parent)
 
 StatusNotifierItem::~StatusNotifierItem()
 {
-    QDBusConnection::sessionBus().unregisterObject("/StatusNotifierItem");
-    QDBusConnection::sessionBus().unregisterService(mService);
-    QDBusConnection::sessionBus().disconnectFromBus(mService);
+    mSessionBus.unregisterObject("/StatusNotifierItem");
```

Comment:
Qt doc says that `QLatin1String` is only useful for "applications that define QT_NO_CAST_FROM_ASCII". But it adds, "If the function you're calling with a QLatin1String argument isn't actually overloaded to take QLatin1String, the implicit conversion to QString will trigger a memory allocation...." And QDBusConnection::unregisterObject() doesn't seem to be overloaded. With all this, isn't "/StatusNotifierItem" better?